### ISSUE TITLE
Launch Day Operation 

### ISSUE THREAD

**Issue by jennijuju** [lotus-web3/dotStorage-deal-renewal#9]
https://docs.google.com/document/d/1ZYWw2TcJuql1xFeCdXfr_idHI_pyDpA-fnlVy_jcm20/edit?usp=sharing

**Comment by elizabeth-griffiths** [comment]
Launch day task: David Choi (can't tag him for some reason) will pull URLs. Currently he will plan to pull around 3pm EST. We need to advise him this week (sooner the better) if we prefer they be pulled sooner, and by when.

**Comment by elizabeth-griffiths** [comment]
Noting Launch is planned for 3pm EST for 14th, assuming mainnet launch goes well. Lucky for me, you all have daylight savings before then so I'll have a bright and early one at 6am after all! :)

**Comment by elizabeth-griffiths** [comment]
Confirmed: David Choi will refresh the URLs the morning of 14 March, and send to you all no later than 12pm EST. (He won't send to me because I'll still be asleep.)

**Comment by elizabeth-griffiths** [comment]
@shrenujbansal i will create a source of truth for SPs that should be whitelisted. 
@brendalee are you able to confirm no later than 10 March if the 11 signed up so far are good to whitelist, and which are PL SPs?

**Comment by jennijuju** [comment]
> @shrenujbansal i will create a source of truth for SPs that should be whitelisted.
> @brendalee are you able to confirm no later than 10 March if the 11 signed up so far are good to whitelist, and which are PL SPs?

Let’s avoid calling them PL SPs. PL doesn’t run any storage provider on the network - theses are Labbers who runs SPs on the side, out of their job.

**Comment by brendalee** [comment]
yes - @elizabeth-griffiths I pinged you on slack. We can add all of them to the list, with the exception of "meest nam" for now (we can add them later if they turn on http retrievals) - assuming the list is mutable.

**Comment by elizabeth-griffiths** [comment]
Runbook from my end:
- Blog post - immediately after launch called success - DRI to post TBD
- (in parallel to next task) DRI Dame - First tweet - once blog live (link to blog) -
- DRI - Elizabeth -Send email and slack message to SPs
- DRI - Elizabeth to slack partners who have agreed to support up the link to our twitter to let them know we’re ready for their support

TBD - When press release will be distributed.

**Comment by elizabeth-griffiths** [comment]
Acutally, I popped it into a chart so that it's easier to follow. Here it is: https://www.notion.so/ecosystem-wg/Launch-day-plan-dd06874d7d5246c3820b9281558d1b59?pvs=4
